x-pack/plugins/security_solution/public/flyout/right/components/user_entity_overview.tsx

```diff
@@ -37,9 +37,14 @@ import { useObservedUserDetails } from '../../../explore/users/containers/users/
 const StyledEuiBetaBadge = styled(EuiBetaBadge)`
   margin-left: ${({ theme }) => theme.eui.euiSizeXS};
 `;
-
+const USER_ICON = 'user';
 const CONTEXT_ID = `flyout-user-entity-overview`;
 
+const StyledEuiLink = styled(EuiLink)`
+  font-size: 12px;
```

Comment:
same comment as in host overview about reusing the style from expandable panel

Reply:
I'm actually using emotion directly now